### ISSUE TITLE
把instant-ngp中编码方式改为hashgrid之后，torsor阶段的预测质量很低是为什么

### ISSUE THREAD

**Issue by zckcola** [yerfor/GeneFace#269]

**Comment by zckcola** [comment]
请问是cuda12版本以上的有问题吗

**Comment by zckcola** [comment]
这是改成hash以后torsor阶段的
![image](https://github.com/yerfor/GeneFace/assets/145427194/4464c53a-fe02-4603-9b66-7f84a9624eac)
这是head的
![image](https://github.com/yerfor/GeneFace/assets/145427194/a3d0b240-5045-4995-8209-c14a7351b5f9)
这是config文件accumulate_grad_batches: 1
ambient_out_dim: 2
amp: true
base_config:
- ./lm3d_radnerf_hash.yaml
binary_data_dir: data/binary/videos
bound: 1
camera_offset:
- 0
- 0
- 0
camera_scale: 4.0
clip_grad_norm: 0
clip_grad_value: 0
cond_out_dim: 64
cond_type: idexp_lm3d_normalized
cond_win_size: 1
cuda_ray: true
debug: false
density_thresh: 10
density_thresh_torso: 0.01
desired_resolution: 2048
dt_gamma: 0.00390625
eval_max_batches: 100
far: 0.9
finetune_lips: true
finetune_lips_start_iter: 200000
geo_feat_dim: 128
grid_interpolation_type: linear
grid_size: 128
grid_type: hashgrid
gui_fovy: 21.24
gui_h: 512
gui_max_spp: 1
gui_radius: 3.35
gui_w: 512
head_model_dir: checkpoints/May/lm3d_radnerf_hash
hidden_dim_ambient: 128
hidden_dim_color: 128
hidden_dim_sigma: 128
individual_embedding_dim: 4
individual_embedding_num: 13000
infer_audio_source_name: ''
infer_bg_img_fname: ''
infer_c2w_name: ''
infer_cond_name: ''
infer_lm3d_clamp_std: 2.5
infer_lm3d_lle_percent: 0.0
infer_lm3d_smooth_sigma: 0.0
infer_out_video_name: ''
infer_scale_factor: 1.0
infer_smo_std: 0.0
infer_smooth_camera_path: true
infer_smooth_camera_path_kernel_size: 7
lambda_ambient: 0.1
lambda_lpips_loss: 0.01
lambda_weights_entropy: 0.0001
load_ckpt: ''
load_imgs_to_memory: false
log2_hashmap_size: 16
lr: 0.0005
max_ray_batch: 4096
max_steps: 16
max_updates: 250000
min_near: 0.05
n_rays: 65536
near: 0.3
num_ckpt_keep: 1
num_layers_ambient: 3
num_layers_color: 2
num_layers_sigma: 3
num_sanity_val_steps: 2
num_steps: 16
num_valid_plots: 5
optimizer_adam_beta1: 0.9
optimizer_adam_beta2: 0.999
print_nan_grads: false
processed_data_dir: data/processed/videos
raw_data_dir: data/raw/videos
resume_from_checkpoint: 0
save_best: true
save_codes:
- tasks
- modules
- egs
save_gt: true
scheduler: exponential
seed: 9999
smo_win_size: 5
smooth_lips: false
task_cls: tasks.radnerfs.radnerf_torso.RADNeRFTorsoTask
tb_log_interval: 100
torso_head_aware: false
torso_individual_embedding_dim: 8
torso_shrink: 0.8
torso_train_mode: 1
update_extra_interval: 16
upsample_steps: 0
use_window_cond: true
val_check_interval: 2000
valid_infer_interval: 10000
valid_monitor_key: val_loss
valid_monitor_mode: min
video_id: May
warmup_updates: 0
weight_decay: 0
with_att: true
work_dir: checkpoints/May/lm3d_radnerf_torso_hash
